Extending/Packages/UmbPack/index.md

```diff
@@ -0,0 +1,174 @@
+There you will find a button under each package to manage API keys:
+
+![Api keys button](images/apiKeysButton.png)
+
+To generate a key, you need to add a description, when you click add it will show you your key. Be aware that as soon as you refresh the page the key will be gone, and if you lose it you will have to generate a new one.
+
+![Example key](images/exampleKey.png)
+
+Once you copy the key you are ready to use it, but first you must set up the UmbPack tool:
+
+## Installing UmbPack locally
+
+To install the latest version of the tool locally you need to run the following command in a commandline:
+
+```
+dotnet tool install --global Umbraco.Tools.Packages
+```
+
+You will then be able to use it by running `umbpack [command]` in the command line afterwards. More on the commands of the tool below.
+
+## The Init command
+
+The init command helps you create a package.xml file, which is the file in an Umbraco package that contains all the metadata for a package - things like author info, version compatibility, etc.
+The init command can be used by typing:
+
+```
+umbpack init
+```
+
+It is usually used to scaffold a package.xml file that you can then use with the `pack` command to create an Umbraco package.
+
+Here is an example of what it will look like in the commandline. It will also post back the package.xml for verification at the end:
+
+![UmbPack init example](images/umbpackinit.png)
+
+## The Pack command
+
+The pack command is used to create an Umbraco zip package from some specified files. It is similar to what would happen if you picked the files via the backoffice.
+
+:::warn
+The pack command has no way of handling package actions, Umbraco schema and content! If you need any of those you will have to still pick them from the backoffice.
+:::
+
+The pack command has a few options, the only mandatory one is to point it at either a package.xml file or a folder containing a package.xml file:
+
+**Example of packing a folder**
+
+```
+umbpack pack C:\Umbraco\Customers\test\Test.Web\App_Plugins\CustomPackage
+```
+
+In this example I wanted to pack up the ~App_Plugins/CustomPackage folder inside the .Web project. Often that is where you will store package files, the tool will ensure it adds all files to the package.xml inside the folder so if you continue to develop and add things a new pack will update it.
+
+**Example of packing based on a package.xml file**
+
+```
+umbpack pack C:\Umbraco\Customers\test\package.xml
+```
+
+In this example you can imagine I keep a package.xml file outside of the project folders, inside the `src` folder of a Visual Studio solution. The benefit here is that it would not be part of the website, but could still be source controlled. Additionally this is a better approach if you want your package to include files outside of a specific folder. 
```

Comment:
```suggestion
In this example you can imagine I keep a `package.xml` file outside of the project folders, inside the `src` folder of a Visual Studio solution. The benefit here is that it would not be part of the website, but could still be source controlled. Additionally this is a better approach if you want your package to include files outside of a specific folder. 
```